lua/nvim_lsp.lua

```diff
@@ -11,6 +11,7 @@ require 'nvim_lsp/gopls'
 require 'nvim_lsp/hie'
 require 'nvim_lsp/leanls'
 require 'nvim_lsp/pyls'
+require 'nvim_lsp/mpls'
```

Comment:
Unless `mpls` is an existing convention, wouldn't it  be more intuitive to call this `mspyls`? 

- We already have `pyls`, so calling this `_pls` is needless variation.
- `ms` is a common abbreviation of "Microsoft"

Reply:
MPLS was in my head from reading this [doc](https://github.com/microsoft/python-language-server/blob/7d0e3eb4752296fea66c92fa2722f77c1c9820ab/Using_in_sublime_text.md) in Microsoft's repo  when trying to get the language server running. I agree with you though, and will rename to pyls_ms.